resources/views/dashboard.blade.php

```diff
@@ -1,3 +1,10 @@
+@php
+    /** @var $feed @mixed */
+    $list_lambda = function ($p) { 
+        return [ 'post' => $p ]; 
```

Comment:
```suggestion
        return [ 'post' => $p, 'user' => $p->user()->first(), 'group' => $p->group()->first() ]; 
```

Reply:
Solved.
I've changed the name of the Controller and  I've done a new view for the home endpoint.